test/e2e/specs/preview.test.js

```diff
@@ -6,23 +6,36 @@ import { parse } from 'url';
 /**
  * Internal dependencies
  */
-import '../support/bootstrap';
 import {
 	newPost,
-	newDesktopBrowserPage,
 	getUrl,
 	publishPost,
 } from '../support/utils';
 
 describe( 'Preview', () => {
-	beforeAll( async () => {
-		await newDesktopBrowserPage();
-	} );
-
 	beforeEach( async () => {
 		await newPost();
 	} );
 
+	async function openPreviewPage( editorPage ) {
+		let openTabs = await browser.pages();
+		expect( openTabs ).toHaveLength( 2 );
```

Comment:
This function seems difficult to maintain, particularly if we add any more test cases, or "resolve" the issue where we have a blank open tab present in the test browser.

Do we really care if there are two tabs?
Couldn't we express the "wait for" as `+ 1` of however many open tabs there are initially?

Reply:
Yes, I had the same feeling when I saw it for the first time.

Reply:
Yeah that's fair, I was trying to check here for the double preview tab issue, but as that's a very rare intermittent issue I'll change this and continue investigating it elsewhere.